java/src/jmri/jmrit/dispatcher/DispatcherBundle.properties

```diff
@@ -345,7 +347,9 @@ Error41 = No Loco was selected,\nplease select a Loco Entry.
 Error42 = No Train was selected,\nplease select a Train.
 Error43 = Train has no valid DCC address,\nplease set it manually.
 Error44 = No Transit was selected,\nplease select a Transit.
-Error45 = Train Name Cannot be Blank
+Error46 = Roster [{0}] not found.
+Error48 = Failed to create Active Train [{0}].\nPlease check system log.
+Error49 = Train Max Throttle {0} must be greater by 10% or more than Minimum Throttle Operating Speed {1} speed.
```

Comment:
You have two conditions here.
- Greater than 10%
- More than the minimum speed.

I assume that both conditions needs to be true but you have "or" instead of "and" between them.